examples/save_disp.py

```diff
@@ -0,0 +1,68 @@
+import os
+from pathlib import Path
+from packaging import version
+
+import torch
+import numpy as np
+from PIL import Image
+
+
+def load_image(imfile):
+    """Utility function to load image files as tensors"""
+    img = Image.open(imfile)
+    img = np.array(img).astype(np.uint8)
+    img = torch.from_numpy(img).permute(2, 0, 1).float()
```

Comment:
I presume there's no normalisation required?
Point being, I guess this is just sort of a copy of what the authors were doing?

Reply:
Normalization is not required. In the examples, the authors provided the mean and standard deviation tensors for the model and [initialized an object for normalizing input data](https://github.com/fizyr/MonSter/blob/main/save_disp.py), but they never used it.

This function is indeed a copy.